app/views/lists/_errors.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="errors">
+  <strong><%= pluralize(errors.count, 'error') %>:</strong>
+  <% intersperse(errors, ", ").each do |msg| %>
+    <span><%= msg %></span>
+  <% end %>
```

Comment:
My gut reaction to your intersperse method is that it is unusual. But I am struggling right now to suggest a better way to do what you're doing in this context. I'll get back to you ;).

Reply:
Yep, I made this more complicated than it needs to be. 😆 I've changed to use `.join(", ")` in e41970e